docs/source/llms.txt.template

```diff
@@ -0,0 +1,186 @@
+- [GPT-2 Training]({{BASE_URL}}/examples/models/gpt-2.html.md): Train GPT-2 from scratch
+
+## Performance & Optimization
+
+- [AWS EFA]({{BASE_URL}}/examples/performance/aws_efa.html.md): High-performance networking on AWS
+- [GCP GPU Direct TCPx]({{BASE_URL}}/examples/performance/gcp_gpu_direct_tcpx.html.md): GPU Direct on GCP
+- [Nebius InfiniBand]({{BASE_URL}}/examples/performance/nebius_infiniband.html.md): InfiniBand on Nebius
+
+## Applications
+
+- [RAG Systems]({{BASE_URL}}/examples/applications/rag.html.md): Build retrieval-augmented generation
+- [Batch Inference]({{BASE_URL}}/examples/applications/batch_inference.html.md): Run batch inference jobs
+- [LocalGPT]({{BASE_URL}}/examples/applications/localgpt.html.md): Deploy LocalGPT
+- [Stable Diffusion]({{BASE_URL}}/examples/applications/stable_diffusion.html.md): Image generation
+- [Tabby]({{BASE_URL}}/examples/applications/tabby.html.md): Code completion service
+- [Vector Databases]({{BASE_URL}}/examples/applications/vector_database.html.md): Deploy vector DBs
+
+## Orchestration
+
+- [Airflow Integration]({{BASE_URL}}/examples/orchestrators/airflow.html.md): Use with Apache Airflow
+- [GitHub Actions]({{BASE_URL}}/examples/orchestrators/github_actions.html.md): CI/CD integration
+- [Cron Jobs]({{BASE_URL}}/examples/orchestrators/cron.html.md): Schedule recurring jobs
+- [Jupyter Notebooks]({{BASE_URL}}/examples/frameworks/jupyter.html.md): Run Jupyter on clusters
+- [DVC Integration]({{BASE_URL}}/examples/frameworks/dvc.html.md): Data version control
+- [MPI]({{BASE_URL}}/examples/frameworks/mpi.html.md): Run MPI applications
+
+## Cloud Providers
+
+- [AWS Setup]({{BASE_URL}}/cloud-setup/cloud-permissions/aws.html.md): Amazon Web Services configuration
+- [GCP Setup]({{BASE_URL}}/cloud-setup/cloud-permissions/gcp.html.md): Google Cloud Platform setup
+- [Kubernetes Setup]({{BASE_URL}}/cloud-setup/cloud-permissions/kubernetes.html.md): Kubernetes cluster configuration
+- [Nebius Setup]({{BASE_URL}}/cloud-setup/cloud-permissions/nebius.html.md): Nebius cloud configuration
+- [vSphere Setup]({{BASE_URL}}/cloud-setup/cloud-permissions/vsphere.html.md): VMware vSphere configuration
+
+## Advanced Topics
+
+- [Comparison]({{BASE_URL}}/reference/comparison.html.md): Compare with other tools
+- [Troubleshooting]({{BASE_URL}}/reference/kubernetes/kubernetes-troubleshooting.html.md): Common issues and solutions
+- [TPU Support]({{BASE_URL}}/reference/tpu.html.md): Using TPUs
+- [Quota Management]({{BASE_URL}}/cloud-setup/quota.html.md): Manage cloud quotas
+
+## Optional
+
+- [Release Notes](https://github.com/skypilot-org/skypilot/releases): Version history
+- [Contributing](https://github.com/skypilot-org/skypilot/blob/master/CONTRIBUTING.md): Contribution guidelines
+- [Community Slack](https://slack.skypilot.co/): Join the community
+- [GitHub Repository](https://github.com/skypilot-org/skypilot): Source code and issues
```

Comment:
We should include the blog page as well `blog.skypilot.co` and probably the community page: https://blog.skypilot.co/community/

We can also include the case studies page: https://blog.skypilot.co/case-studies/